lib/middleman-simple-thumbnailer/extension.rb

```diff
@@ -37,12 +37,35 @@ def after_build(builder)
       def image_tag(path, options={})
         resize_to = options.delete(:resize_to)
         return super(path, options) unless resize_to
+        ext = app.extensions[:middleman_simple_thumbnailer]
 
-        image = MiddlemanSimpleThumbnailer::Image.new(path, resize_to, app)
         if app.development?
-          super("data:#{image.mime_type};base64,#{image.base64_data}", options)
+
+          # For better performance on local development
+          # (high bandwidth)
+          # simply change width in <img>-tag
+          
+          original_width  = options.delete(:width)
+          resized_width   = resize_to.split('x').first.to_i
+          original_height = options.delete(:height)
+          resized_height  = resize_to.split('x').last.to_i
+          
+          if !original_width.nil? && original_width <= resized_width
+            if original_height <= resized_height
+              options[:width] = original_width
```

Comment:
Ok. I understand that changing the width and height is not a sound solution. Could you explain why the image is base64-included in the first place? Is that because the cached file is not available to the web-client?